tune_sklearn/tune_gridsearch.py

```diff
@@ -259,7 +259,9 @@ def _tune_run(self, config, resources_per_trial):
             resources_per_trial=resources_per_trial,
             local_dir=os.path.expanduser(self.local_dir),
             loggers=self.loggers,
-            time_budget_s=self.time_budget_s)
+            time_budget_s=self.time_budget_s,
+            metric=self._metric_name,
+            mode="max")
```

Comment:
hm is it possible to make 'mode' configurable?

Reply:
I guess it is not a regression?

Reply:
(feel free to resolve)

Reply:
We could make it configurable, shouldn't be an issue